docs/src/arrays.md

```diff
@@ -175,14 +175,14 @@ These identities are particularly useful:
 \begin{align*}
 \frac{d}{dt} \left( \Re(A) \right) &= \Re(\dot{A})\\
 \frac{d}{dt} \left( A^* \right) &= \dot{A}^*\\
-\frac{d}{dt} \left( A^T \right) &= \dot{A}^T\\
-\frac{d}{dt} \left( A^H \right) &= \dot{A}^H\\
+\frac{d}{dt} \left( A^\mathrm{T} \right) &= \dot{A}^\mathrm{T}\\
+\frac{d}{dt} \left( A^\mathrm{H} \right) &= \dot{A}^\mathrm{H}\\
```

Comment:
I am included towards `\mathsf` per https://tex.stackexchange.com/a/217623/5834
```suggestion
\frac{d}{dt} \left( A^\mathsf{T} \right) &= \dot{A}^\mathsf{T}\\
\frac{d}{dt} \left( A^\mathsf{H} \right) &= \dot{A}^\mathsf{H}\\
```

Should we define a  macro for this in our config?
I am included towards not, while it is possible it seems annoying top do, and confusing to the reader?
I think it is something like the following
```
format = Documenter.HTML(mathengine=KaTeX(Dict(
    :macros=>[
        raw"\H"=>raw"\mathsf{H}",
        raw"\T"=>raw"\mathsf{T}",
    ]
)))
```

Reply:
Likewise, yeah, I think we should avoid macros in general to avoid confusion. The ones I've added in this PR are copied from the physics package, and when we  resolve #288 they can be deleted.
The `\mathsf` suggestion is good.